src/test.ts

```diff
@@ -13,7 +13,9 @@ declare const require: any;
 // First, initialize the Angular testing environment.
 getTestBed().initTestEnvironment(
     BrowserDynamicTestingModule,
-    platformBrowserDynamicTesting(),
+    platformBrowserDynamicTesting(), {
+        teardown: { destroyAfterEach: false },
```

Comment:
why not, isn't that the safe way ?

Reply:
supprimer tout les composants après chaque tests, c'est pas teardown.destroyAfterEach: true, du coup ?

Reply:
abawi! fixed